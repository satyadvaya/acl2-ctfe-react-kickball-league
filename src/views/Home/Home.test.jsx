import { render } from '@testing-library/react';
import Home from './Home';

it('should render a simple homepage', () => {
  const { container } = render(<Home />);
  expect(container).toMatchSnapshot();
});
