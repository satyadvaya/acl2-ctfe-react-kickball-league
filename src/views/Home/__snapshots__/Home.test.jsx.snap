// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render a simple homepage 1`] = `
<div>
  <p>
    Home
  </p>
</div>
`;
